editor/editor_node.cpp

```diff
@@ -5137,16 +5137,36 @@ void EditorNode::_scene_tab_closed(int p_tab) {
 	scene_tabs->update_scene_tabs();
 }
 
-Button *EditorNode::add_bottom_panel_item(String p_text, Control *p_item) {
-	Button *tb = memnew(Button);
+class EditorBottomDockButton : public Button {
+	GDCLASS(EditorBottomDockButton, Button)
+
+	static void _bind_methods() {
+		ADD_SIGNAL(MethodInfo("dropping"));
+	}
+
+public:
+	virtual bool can_drop_data(const Point2 &p_point, const Variant &p_data) const override {
+		if (!is_pressed()) {
+			const_cast<EditorBottomDockButton *>(this)->emit_signal("dropping");
+		}
+		return true;
```

Comment:
```suggestion
		return false;
```
`true` does not make sense.